src/onegov/org/views/people.py

```diff
@@ -26,7 +26,18 @@ def view_people(
     layout: PersonCollectionLayout | None = None
 ) -> 'RenderData':
 
-    people = self.query().order_by(Person.last_name, Person.first_name).all()
+    selected_org = request.params.get('organisation', None)
+    selected_sub_org = request.params.get('sub_organisation', None)
+
+    query = self.query().order_by(Person.last_name, Person.first_name)
```

Comment:
We normally implement filters in the collection, see for example UserCollection, TicketCollection etc. This way, you can ensure it's working correctly with pagination and also use the filters in other instances (e.g. API)